sequence_task_3/student/src/impl.cpp

```diff
@@ -1,2 +1,20 @@
 #include "impl.h"
 
+std::string reverseWords(const std::string& input)
+{
+    std::istringstream input_stream(input);
+    std::deque<std::string> words;
+    std::move(std::istream_iterator<std::string>(input_stream),
```

Comment:
Nice try, but when dereferenced, const lvalue reference is returned for istream iterator:
https://en.cppreference.com/w/cpp/iterator/istream_iterator/operator*